shell/run_tests.sh

```diff
@@ -114,20 +114,9 @@ set -x
 # Setup and run tests.
 #
 
-#REFRESH_CACHE_ON_FETCH="$(jq -r ".skip_cache" $BUILD_JSON)"
-#REFRESH_CACHE_ON_WRITE="$(jq -r ".rebuild_cache" $BUILD_JSON)"
-#JEDI_BUNDLE_BRANCH="$(jq -r ".jedi_bundle_branch" $BUILD_JSON)"
-
 # Extract just the repo name from the full repository path
 TRIGGER_REPO=$(echo "$TRIGGER_REPO_FULL" | cut -d'/' -f2)
 
-
-# Generate the version ref flag value used later for build config. Ignore
-# entries with null version_ref.commit values they are branch references already
-# configured in the bundle.
-UNIT_DEPENDENCIES=$(jq -r '.dependencies | join(" ")' $BUILD_JSON)
-
-
```

Comment:
Note the logic on line 200 (left side of diff) is why this breaks oasim but mostly leaves other pull requests unaffected (although I think its also breaking mpas-jedi but I'm not entirely sure).